### PR TITLE
Ecommerce 2020 queries

### PR DESCRIPTION
**Pull request by jrharalson**
Progress on #914 

Initial check-in PR/placeholder

### CODE REVIEW

**Comment by rviscomi** [comment]
Hey @jrharalson. I know this isn't marked for review yet, but I wanted to give you a quick heads up about something. There's a new SQL naming convention this year where we'll be naming the queries more descriptively based on what they're analyzing as opposed to chapter/metric numbers. So for example `13_01.sql` could be `top_vendors.sql`. This makes them more discoverable later.

**Comment by rviscomi** [comment]
Be sure to update this from "Draft" to "Ready for review" so we can get more eyes on it

File: sql/2020/16_Ecommerce/top_cdn_bydevice_vendor_wapp.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+#standardSQL
+# 13_15b: % of CDN usage for eComm - solely from Wapp
+SELECT
+  vendor,
+  COUNTIF(category = 'CDN') AS Cdnfreq,
+  COUNT(0) AS total,
+  ROUND(COUNTIF(category = 'CDN') * 100 / COUNT(0), 2) AS pct
+  FROM
+    `httparchive.sample_data.technologies_*`
```
Comment (rviscomi) [inline sql/2020/16_Ecommerce/top_cdn_bydevice_vendor_wapp.sql:9]: And group by `_TABLE_SUFFIX`
```suggestion
    `httparchive.technologies.2020_08_01_*`
```

**Review by rviscomi — CHANGES_REQUESTED** [review]

**Review by jrharalson — COMMENTED** [review]
Thanks for the feedback. Made a few updates and push a few updates that were stuck pending a check-in. Need to rework the LCP sql later.

**Comment by rviscomi** [comment]
@jrharalson sounds good let me know when this is ready for another review

File: sql/2020/16_Ecommerce/lcp_byvendor_bydevice.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#standardSQL
+# 13_08: CrUX LCP performance of ecommerce providers
+#standardSQL
+# 13_08: CrUX LCP performance of ecommerce providers
+SELECT
+    client,
+    ecomm,
+    AVG(ROUND(SAFE_DIVIDE(fast_lcp, fast_lcp + avg_lcp + slow_lcp) * 100, 2)) AS fast,
```
[the remark this answers is not in the document]
Comment (rviscomi) [reply sql/2020/16_Ecommerce/lcp_byvendor_bydevice.sql:6]: The second suggestion can be easily adapted from the [CMS chapter's query](https://github.com/HTTPArchive/almanac.httparchive.org/blob/7d373c3fa1ba0b3d78ed41484115552a52e4350e/sql/2020/15_CMS/core_web_vitals_passing.sql#L51) by replacing `WHERE category = 'CMS'` with `'Ecommerce'`. This gives you LCP, FID, CLS, and the overall CWV assessment.

You can also adapt [this query](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/15_CMS/core_web_vitals_distribution.sql) of good/ni/poor speeds for each metric.

**Comment by rviscomi** [comment]
@jrharalson how's this coming along?

**Comment by jrharalson** [comment]
Updated to the latest Crux style

File: sql/2020/16_Ecommerce/pagestats_image_bydevice_format.sql [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+SELECT
+  client,
+  format,
+  COUNT(0) AS freq,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
+  ROUND(COUNT(0) * 100 / SUM(COUNT(0)) OVER (PARTITION BY client), 2) AS pct
+FROM
+  `httparchive.almanac.summary_requests`
+JOIN
+  (SELECT _TABLE_SUFFIX AS client, url AS page FROM `httparchive.technologies.2020_07_01_*` WHERE category = 'Ecommerce')
```
Comment (rviscomi) [inline sql/2020/16_Ecommerce/pagestats_image_bydevice_format.sql:12]:
```suggestion
  (SELECT _TABLE_SUFFIX AS client, url AS page FROM `httparchive.technologies.2020_08_01_*` WHERE category = 'Ecommerce')
```

**Review by rviscomi — APPROVED** [review]
Just one small correction, then this should be good to go

**Review by rviscomi — APPROVED** [review]

**Pull request merged**